benchmarks/microbenchmarks/_utils.py

```diff
@@ -0,0 +1,47 @@
+from taichi._lib import core as ti_core
+
+import taichi as ti
+
+
+def _size2tag(size_in_byte):
+    size_subsection = [(0.0, 'B'), (1024.0, 'KB'), (1048576.0, 'MB'),
+                       (1073741824.0, 'GB'),
+                       (float('inf'), 'INF')]  #B KB MB GB
+    for dsize, unit in reversed(size_subsection):
+        if size_in_byte >= dsize:
+            return str(int(size_in_byte / dsize)) + unit
+
+
+def tags2name(tag_list):
+    name = ''
+    for tag in tag_list:
+        name += '_' + tag
+    return name.lstrip("_")
```

Comment:
```suggestion
    return '_'.join(tag_list)
```

Reply:
`'_'.join(tag_list)` should work exactly as the function here?

Reply:
my bad, verified locally but missed it when submitting 😭